dashboard/test/controllers/api/v1/pd/workshop_attendance_controller_test.rb

```diff
@@ -1,6 +1,6 @@
 require 'test_helper'
 
-class Api::V1::Pd::WorkshopAttendanceControllerTest < ::ActionController::TestCase
+class Api::V1::Pd::WorkshopAttendanceControllerTest < ::ActionDispatch::IntegrationTest
```

Comment:
I'm not sure at this point - Rails 5 has deprecated ActionController entirely, so all of the tests should be migrated to ActionDispatch::IntegrationTest eventually. I'm not sure what the implications of that will be on our current code folder structure, or what the new folder conventions/recommendations are. Probably something that can be addressed after this initial upgrade PR.